src/test/controller/BaseController.ts

```diff
@@ -28,7 +29,10 @@ export class BaseController {
   constructor() {
     this.mockRedisClient = BaseController.createRedisMock();
     this.mockFirebaseAdmin = BaseController.createFirebaseMock();
-    this.mockGeoCoder = BaseController.createMockNodeGeocoder();
+    const mockGeocoder = BaseController.createMockNodeGeocoder();
+    this.mockGeocoderConfig = {
+      geocoder: mockGeocoder,
+    };
```

Comment:
I was thinking something like this to stay more consitent with what we have
```suggestion
    this.mockGeocoderConfig = BaseController.createMockNodeGeocoderConfig();
```